src/postamble.js

```diff
@@ -149,7 +149,7 @@ function callMain(args) {
   args = args || [];
 
   var argc = args.length+1;
-  var argv = stackAlloc((argc + 1) * {{{ Runtime.POINTER_SIZE }}});
+  var argv = stackAlloc((argc + 1) * {{{ `${Runtime.POINTER_SIZE}` }}});
```

Comment:
Sorry, I asked this in the big PR:

https://github.com/emscripten-core/emscripten/pull/12869#discussion_r721694480

The eval will turn the string back into an integer, I guess, but there is no need.